app/controllers/api/articles_controller.rb

```diff
@@ -4,13 +4,30 @@ class Api::ArticlesController < ApplicationController
   before_action :authenticate_user!, only: [:create]
 
   def index
-    article = Article.where(published: true)
-    render json: article, each_serializer: Article::IndexSerializer
+    check_index_params
+    category = params[:category] || 'all'
+
+    case category
+    when 'local'
+      articles = find_articles({}, { id: -1 })
+    when 'current'
+      last_24hrs = Time.now - 1.day..Time.now
```

Comment:
what is 2 dots?

Reply:
That's the range syntax. We search for articles with date between 'now minus one day' and 'now', so everything from the last 24 hours.